extensions/amp-pinterest/0.1/test/test-amp-pinterest.js

```diff
@@ -57,4 +60,38 @@ describes.realWin('amp-pinterest', {
         'scription=Next%20stop%3A%20Pinterest');
     });
   });
+
+  it('sets provided alternate text', () => {
+    return getPin('embedPin',
+        'https://www.pinterest.com/pin/99360735500167749/',
+        null,
+        null,
+        'Hands making heart over Pinterest logo'
+    ).then(pin => {
+      const img = pin.querySelector('img');
+      img.getAttribute('alt').to.equal('Hands making heart over ' +
+        'Pinterest logo');
+    });
+  });
+
+  it('sets alternate text from pin data provided by Pinterest API', () => {
```

Comment:
For this and the next one to work, the `env.fetchMock` needs to be setup to return a mock JSON data similar to what pinterest API would return. We mock the `fetch` so we never endup actually making a network request. 

I suggest using differ PIN urls for this and next tests and use `getOnce(url, response)` to mock a response for each of those urls ( note that Url will be the API call Url e.g. `https://widgets.pinterest.com/v3/pidgets/pins/info/`pin_ids=${PIN-ID}&sub=www&base_scheme=https`